io-backend/src/main/java/agh/io/iobackend/model/GameRoom.java

```diff
@@ -0,0 +1,40 @@
+package agh.io.iobackend.model;
+
+import javax.persistence.*;
+
+@Entity(name = "GameRoom")
+public class GameRoom {
+    @Id
+    @SequenceGenerator(
+            name = "gameRoom_sequence",
+            sequenceName = "gameRoom_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "gameRoom_sequence"
+    )
+    @Column(
+            name = "gameRoomID",
+            updatable = false
+    )
+    private Long gameRoomID;
+
+    @Column(
+            name = "gameMasterID",
+            nullable = false
+    )
+    private Long gameMasterID;
+
+    @Column(
+            name = "mapID"
+    )
+    private Long mapID;
+
+    @Column(
+            name = "roomCode",
+            unique = true,
+            nullable = false
+    )
+    private String roomCode;
```

Comment:
Można by tu jeszcze dodać parametry typu czas rundy, maksymalna liczba graczy - to co można wybrać tworząc pokój

Reply:
Dodałem oba te parametry. Po przejściu makiety nie przyszło mi do głowy nic więcej co mogło by być nam potrzebne. Okomentowałem też atrybuty z bazy